services/src/main/java/org/keycloak/authorization/authorization/AuthorizationTokenService.java

```diff
@@ -469,9 +501,9 @@ private EvaluationContext createEvaluationContext(KeycloakAuthorizationRequest r
     }
 
     private void resolvePreviousGrantedPermissions(PermissionTicketToken ticket,
-            KeycloakAuthorizationRequest request, ResourceServer resourceServer,
-            Map<String, ResourcePermission> permissionsToEvaluate, ResourceStore resourceStore, ScopeStore scopeStore,
-            AtomicInteger limit) {
+                                                   KeycloakAuthorizationRequest request, ResourceServer resourceServer,
```

Comment:
Could you please, remove changes that are basically formatting the code?

Reply:
agreed